fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/bitmap/ComposedBitmapIndexAggregate.java

```diff
@@ -88,7 +88,7 @@
     @Nonnull
```

Comment:
Is the description in line 85 still accurate given that the new Function does not consider ordering of expressions?

Reply:
Will change the javadoc. This logic here should take a call and bind it. It's one of the TODOs left. If you look at how it's used downstream, the caller binds the call to a function instead of going through this path.